types/is-object/index.d.ts

```diff
@@ -2,10 +2,11 @@
 // Project: https://github.com/ljharb/is-object
 // Definitions by: Wilson Hobbs <https://github.com/wbhob>
 //                 Jordan Harband <https://github.com/ljharb>
+//                 Minh-Phuc Tran <https://github.com/phuctm97>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 export = isObject;
 
-declare function isObject(a: any): boolean;
+declare function isObject(value: unknown): value is Record<string | symbol | number, unknown>;
```

Comment:
this is missing bigint

Reply:
bigint is not a valid key type, a reference example https://www.typescriptlang.org/play?target=7#code/MYewdgzgLgBA1gUwJ4wLwwEIEsDmBJMKACgCIBOABgoHYBGMsgJgFYAWa1ih2kgSgChQkWCABGAKzQwA3gG1ESALoAuGLQC+AbkHgIIADYIAdPpA4iY8b01A

Reply:
ah, i forgot typescript is still broken in this respect